src/components/file/file.tsx

```diff
@@ -121,6 +121,7 @@ export class File {
                 onDrop={this.handleFileDrop}
                 onDragEnter={this.preventAndStop}
                 onDragOver={this.preventAndStop}
+                leadingIcon={'upload_to_cloud'}
```

Comment:
We probably need to do it like that `leadingIcon="upload_to_cloud"`

Reply:
Either ought to work I think. `leadingIcon="upload_to_cloud"` keeps the whole thing as html, whereas `leadingIcon={'upload_to_cloud'}` switches over to javascript for the value. The end result should be the same.